src/display/editor/tools.js

```diff
@@ -1029,6 +1031,8 @@ class AnnotationEditorUIManager {
     if (this.#mode !== AnnotationEditorType.HIGHLIGHT) {
       return;
     }
+
+    this.showAllEditors("highlight", true, /* updateButton = */ true);
```

Comment:
Nit: Can we please add the blank line *after* instead, since this isn't related to the Shift handling below?
```suggestion
    this.showAllEditors("highlight", true, /* updateButton = */ true);
    
```